source/develop/release-management/features/virt/bochs-display.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Bochs-display
+category: feature
+authors: smelamud
+feature_name: Bochs-display
+feature_modules: engine
+feature_status: In development
+---
+
+# Bochs-display
+
+## Summary
+
+This feature adds new display type `bochs-display` and enables it for
+UEFI VMs.
+
+## Owner
+
+* Name: Shmuel Melamud
+* Email: smelamud@redhat.com
+
+## Description
+
+The bochs-display device is a simple display device which is similar to
+VGA but without legacy emulation. The code size and complexity needed to
+emulate this device is an order of magnitude smaller, resulting in a
+reduced attack surface. Another nice feature is that you can place this
+device in a PCI Express slot.
+
+For UEFI guests it is safe to use the bochs display device instead of
+the standard VGA device. The firmware will setup a linear framebuffer as
+GOP anyway and never use any legacy VGA features.
+
+For BIOS guests this device might be useable as well, depending on
+whenever they depend on direct VGA hardware access or not.
+
+## Changes to Engine
+
+1. Add new `bochs` value to `DisplayType` enum and the corresponding
+   `VmDeviceType.BOCHS` value. This automatically adds 'bochs' to the
+   selection of display types in the UI and enables generation of the
+   correct video device in libvirt XML.
```

Comment:
should this be limited to cluster-level 4.5?
otherwise we'll need convert it, I guess to vga, when exporting VM from CL<4.5